docs/api/desktop-capturer.md

```diff
@@ -94,6 +100,23 @@ which can detected by [`systemPreferences.getMediaAccessStatus`][].
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
 [`systemPreferences.getMediaAccessStatus`]: system-preferences.md#systempreferencesgetmediaaccessstatusmediatype-windows-macos
 
+### `desktopCapturer.isDisplayMediaSystemPickerAvailable()` _Experimental_
+
+Returns `Boolean`, whether or not requesting desktop content via
+the system picker is supporter on this platform.
+
+Currently this will only return `true` on macOS 14.4 and higher. When
+true you should respect this value and use `systemPickerVideoSource` as
+otherwise the OS may present scary warning dialogs to your users.
```

Comment:
Does this mean that we'll recommend to avoid using `desktopCapturer.getSources` in general on macOS starting with macOS 15?

Reply:
That is correct, but don't feel comfortable documenting that till we've tested this new API a bit (you may notice I flagged it as experimental)

Reply:
Sounds good, will keep that in mind!